api/types/src/event_key.rs

```diff
@@ -4,6 +4,15 @@
 use serde::{de::Error as _, Deserialize, Deserializer, Serialize, Serializer};
 use std::{fmt, str::FromStr};
 
+/// Event key is a global index for an event stream.
+///
+/// It is hex-encoded BCS bytes of `EventHandle` `guid` field value, which is
+/// a combination of a `uint64` creation number and account address (without
+/// trimming leading zeros).
+///
+/// For example, event key `0x000000000000000088fbd33f54e1126269769780feb24480428179f552e2313fbe571b72e62a1ca1` is combined by the following 2 parts:
+/// 1. `0000000000000000`: `uint64` representation of `0`.
+/// 2. `88fbd33f54e1126269769780feb24480428179f552e2313fbe571b72e62a1ca1`: 32 bytes of account address.
```

Comment:
This is sort of confusing I feel, is it really the hex encoded bytes of these two? I thought it was just something like:

```
/// Event key is a global index for an event stream.
///
/// An event key is the concatenation of two pieces of information:
///   - Creation number
///   - Account address
///
/// For example, the event key `0x000000000000000088fbd33f54e1126269769780feb24480428179f552e2313fbe571b72e62a1ca1` is the concatenation of the following 2 parts:
/// 1. `0000000000000000`: `uint64` representation of the creation number `0`.
/// 2. `88fbd33f54e1126269769780feb24480428179f552e2313fbe571b72e62a1ca1`: Hex representation of a 32 byte account address.
///
/// Note: The `guid` field of `EventHandle` is an `EventKey`.
```

Reply:
This is even more confusing, because the creation number is little endian, so i'll fix the example